docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md

```diff
@@ -0,0 +1,212 @@
+---
+id: automation-anywhere
+title: Automation Anywhere Connector
+description: Orchestrate your Automation Anywhere queue items from your BPMN process.
+---
+
+## Prerequisites
+
+The **Automation Anywhere Connector** allows you to orchestrate an Automation Anywhere queue from your BPMN process with [Automation Anywhere RPA](https://www.automationanywhere.com/).
+
+To start using the Connector, you must have a running Automation Anywhere instance with a [license](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/control-room/dashboards/cloud-administration-licenses.html) we configured API service. Refer to the [official documentation page](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/aae-client/bot-creator/using-the-workbench/cloud-install.html) to learn more about installing and configuring the Automation Anywhere API service.
```

Comment:
"To start using the Connector, you must have a running Automation Anywhere instance with a [license](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/control-room/dashboards/cloud-administration-licenses.html) we configured API service."

What do you mean here? You need an instance with a license we configured via an API service?

Reply:
Thank you!